server/server.go

```diff
@@ -622,6 +624,44 @@ func (server *Server) itemEntries() (entries []protocol.ItemEntry) {
 	return
 }
 
+// ashyBiome represents a biome that has any form of ash.
+type ashyBiome interface {
+	// Ash returns the ash of the biome.
+	Ash() float64
+	// WhiteAsh returns the white ash of the biome.
+	WhiteAsh() float64
```

Comment:
Could maybe merge this into a single `Ash() (float64, float64)`.

Reply:
Merged.